en/docs/install-and-setup/setup/mi-setup/deployment/deploying_wso2_ei.md

```diff
@@ -39,11 +39,11 @@ When the nodes in the cluster need to communicate with each other, the Micro Int
 
 1.  Create a database named `clusterdb`.
 
-    - [Setting up a MySQL database](../../../setup/databases/setting-up-MySQL)
-    - [Setting up an MSSQL database](../../../setup/databases/setting-up-MSSQL)
-    - [Setting up an Oracle database](../../../setup/databases/setting-up-Oracle)
-    - [Setting up a Postgre database](../../../setup/databases/setting-up-PostgreSQL)
-    - [Setting up an IBM database](../../../setup/databases/setting-up-IBM-DB2)
+    - [Setting up a MySQL database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-MySQL)
+    - [Setting up an MSSQL database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-MSSQL)
+    - [Setting up an Oracle database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-Oracle)
+    - [Setting up a Postgre database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-PostgreSQL)
+    - [Setting up an IBM database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-IBM-DB2)
```

Comment:
As these page names have capital letters, these links will not work when the fix goes live.